Common/Product/SharedProject/Wpf/Commands.cs

```diff
@@ -76,7 +76,11 @@ public static class Commands {
             if (!tb.AcceptsReturn) {
                 var path = e.Parameter as string ?? tb.GetValue(TextBox.TextProperty) as string;
                 while (!string.IsNullOrEmpty(path) && !Directory.Exists(path)) {
-                    path = Path.GetDirectoryName(path);
+                    try {
+                        path = Path.GetDirectoryName(path);
+                    } catch {
```

Comment:
Catch `ArgumentError` here, otherwise next time we run this through FxCop it'll shout at us :)